### PR TITLE
Issue #19 renew token error

### PR DESCRIPTION
**Pull request by pierre-duchene**
After fix on Renew token, the Alamofire requests which need auth, was launch twice.
This is because "buildRequest" method not only build the request but also send it.

So I modify sendAuthenticatedRequest method signature to pass directly request parameters and so build it at very last time.

### CODE REVIEW

File: Sources/NetworkStack.swift [hunk 1]
```diff
@@ -350,36 +350,44 @@ extension NetworkStack {
   fileprivate func sendRequest<T: DataResponseSerializerProtocol>(requestParameters: RequestParameters,
                                queue: DispatchQueue = DispatchQueue.global(qos: .default),
                                responseSerializer: T) -> Observable<(HTTPURLResponse, T.SerializedObject)> {
-    guard let request = self.buildRequest(requestParameters: requestParameters) else {
-      return Observable.error(NetworkStackError.requestBuildFail)
-    }
-    
     if requestParameters.needsAuthorization {
-      return self.sendAuthenticatedRequest(request: request, queue: queue, responseSerializer: responseSerializer)
+		return self.sendAuthenticatedRequest(requestParameters: requestParameters, queue: queue, responseSerializer: responseSerializer)
```
Comment (AliSoftware) [inline Sources/NetworkStack.swift]: Use spaces for indentation to be consistent with the rest of the code

File: Sources/NetworkStack.swift [hunk 2]
```diff
@@ -350,36 +350,44 @@ extension NetworkStack {
...
     } else {
+		guard let request = self.buildRequest(requestParameters: requestParameters) else {
+			return Observable.error(NetworkStackError.requestBuildFail)
+		}
```
Comment (AliSoftware) [inline Sources/NetworkStack.swift]: Same everywhere of course

**Review by AliSoftware — COMMENTED** [review]
Other than the tabs vs spaces, LGTM.

**Comment by pierre-duchene** [comment]
My indent config was wrong :/
I setup to 2 spaces instead of 2 tabs.
I also setup my config to trim whitespaces at the end of lines

**Comment by AliSoftware** [comment]
👌

**Pull request merged**